src/binary-reader-objdump.cc

```diff
@@ -1711,14 +1704,22 @@ Result BinaryReaderObjdump::BeginGlobal(Index index, Type type, bool mutable_) {
 }
 
 void BinaryReaderObjdump::PrintInitExpr(const InitExpr& expr,
-                                        bool as_unsigned) {
-  assert(expr.insts.size() > 0);
+                                        bool as_unsigned,
+                                        bool with_prefix) {
+  if (with_prefix) {
+    PrintDetails(" - init ");
+  }
+
+  if (expr.insts.empty()) {
+    PrintDetails("malformed\n");
```

Comment:
Do we do this kind of things elsewhere in this file?   How about "<MALFORMED>" to be extra clear?

Reply:
Sounds good. I went with `<EMPTY>` to be safe -- on further review I don't think an empty const expression is malformed, just invalid.